### PR TITLE
Git repo helper text

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/RHTAPBUGS-67

## Description

Update validation text for git repo access

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2023-05-11 at 5 11 14 PM
Screenshot 2023-05-11 at 5 11 25 PM

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/ImportForm/SourceSection/SourceSection.tsx [hunk 1]
```diff
@@ -162,7 +164,9 @@ const SourceSection: React.FC<SourceSectionProps> = () => {
           label={label}
           validated={!isValid && ValidatedOptions.error}
           isRequired
-          labelIcon={<HelpPopover bodyContent="Make sure the URL is correct." />}
+          labelIcon={
+            <HelpPopover bodyContent="Make sure the URL is correct. If the repository is private, provide authorization." />
```
Comment (rohitkrai03) [inline src/components/ImportForm/SourceSection/SourceSection.tsx:168]: We do not have authorization options for private repo. This is not a valid help text.

File: src/components/ImportForm/SourceSection/SourceSection.tsx [hunk 2]
```diff
@@ -110,7 +110,9 @@ const SourceSection: React.FC<SourceSectionProps> = () => {
         ) {
           setValidated(ValidatedOptions.error);
           setFieldValue('source.isValidated', false);
-          setHelpTextInvalid('Unable to access repository');
+          setHelpTextInvalid(
+            "We can't access your repository, probably because it's either private or the URL is incorrect.",
```
Comment (rohitkrai03) [inline src/components/ImportForm/SourceSection/SourceSection.tsx:114]: This help text won't be shown if the URL is incorrect.

**Comment by abhinandan13jan** [comment]
cc: @abigaeljamie @beaumorley Let us know, what is to be done?

**Comment by beaumorley** [comment]
Since we do not have authorization functionality yet, I would just remove the question mark and popover text for now.
FYI @abigaeljamie

**Review by abigaeljamie — APPROVED** [review]

**Comment by abhinandan13jan** [comment]
https://github.com/openshift/hac-dev/pull/663

**Pull request closed without merging**